### PR TITLE
Benchmarks for busboy

### PR DESCRIPTION
**Pull request by Uzlopak**
Tadaaa!

I modified the dicer benchmark for busboy. I also added the old busboy for comparison, but i did not add a npm package.json script for benching against the old busboy. I can add it if you like. 

I moved the dicer benchmarks into the bench folder. Looks clear. ;)

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: bench/fastify-busboy-bench.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+for (let i = 0, il = 10; i < il; i++) {
+  const boundary = '-----------------------------168072824752491622650073',
+    d = new Busboy({
+      headers: {
+        'content-type': 'multipart/form-data; boundary=' + boundary
+      }
+    }),
+    mb = 100,
+    buffer = createMultipartBuffer(boundary, mb * 1024 * 1024),
```
Comment (kibertoad) [inline bench/fastify-busboy-bench.js:24]: we probably should extract shared part somewhere to reduce risk of benchmarks diverging over time.

Comment (Uzlopak) [reply bench/fastify-busboy-bench.js:24]: For Fastify-Busboy and Old-Busboy?

Comment (kibertoad) [reply bench/fastify-busboy-bench.js:24]: at the very least, but I assume there is lots of duplication across all benchmarks (which we'll need revise eventually too).

Comment (Uzlopak) [reply bench/fastify-busboy-bench.js:24]: Could we do it in another PR? Benchmarks against other Projects has to be any way cleaned up.

**Comment by Uzlopak** [comment]
btw. The comparison:

```
uzlopak@uzlopak-Lenovo-Legion-5-17ARH05H:~/Workspace/fastify/busboy$ node ./bench/busboy-bench.js 
2380.95 mb/sec
2222.22 mb/sec
3448.28 mb/sec
3571.43 mb/sec
3571.43 mb/sec
3448.28 mb/sec
3448.28 mb/sec
3571.43 mb/sec
3571.43 mb/sec
3448.28 mb/sec

uzlopak@uzlopak-Lenovo-Legion-5-17ARH05H:~/Workspace/fastify/busboy$ node ./bench/fastify-busboy-bench.js 
8333.33 mb/sec
11111.11 mb/sec
10000.00 mb/sec
10000.00 mb/sec
10000.00 mb/sec
10000.00 mb/sec
11111.11 mb/sec
11111.11 mb/sec
10000.00 mb/sec
10000.00 mb/sec
```

File: bench/fastify-busboy-bench.js (hunk 1 above)
Comment (kibertoad) [reply bench/fastify-busboy-bench.js:24]: sure!

**Review by kibertoad — APPROVED** [review]

**Pull request merged**